examples/solid/start-i18n-paraglide/README.md

```
@@ -0,0 +1,196 @@
+# TanStack Start example
+
+This example shows how to use Paraglide with TanStack Start. The source code can be found [here](https://github.com/opral/monorepo/tree/main/inlang/packages/paraglide/paraglide-js/examples/tanstack-start).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)